index.html

```diff
@@ -237,10 +238,18 @@ <h3><span></span>Skills</h3>
               </span>
             </div>
             <div>
-              <form action="">
-                <input type="text" id="fname" name="fname" placeholder="Full name">
-                <input type="text" id="email" name="email" placeholder="Email address">
-                <textarea name="" id="" cols="10" rows="5" placeholder="Enter text here"></textarea>
+              <form action="https://formspree.io/f/xqkjbgbd"  method="post">
+                <ul>
+                  <li>
+                    <input type="text" id="fname" name="fname" placeholder="Full name" maxlength="30" required>
+                  </li>
+                  <li>
+                    <input type="email" id="email" name="email" placeholder="Email address" required>
+                  </li>
+                  <li>
+                    <textarea name="textarea" id="textarea" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+                  </li>
+                </ul>
                 <input type="submit" value="Get in touch">
               </form> 
```

Comment:
- [ ] You should stick to the design templates in Figma as much as possible, So please remove the black border So it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/194014508-36691642-7207-4532-8dac-e37b22c766f8.png)
### Fimga design
![figma](https://user-images.githubusercontent.com/81727988/194014548-4598496d-fa14-411d-ba0c-d422767232bf.png)